ls1intum/ArTEMiS
# Feature/extend tutor dashboard with modeling exercises

### Checklist

### Motivation and Context
With growing numbers of students that use ArTEMiS an increasing number of tutors is necessary to assess the students' submissions. All these tutors have different levels of experience and knowledge. In order to increase the assessment quality and to decrease the number of conflicting assessments for modeling exercises, we want to improve the training of tutors by a tutor dashboard including assessment tutorials.

### Description
We extend the existing tutor dashboard to also support modeling exercises. In the dashboard tutors have to read the grading instructions and review or assess example submissions created by the instructor before being able to assess real student submission.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Create an exercise with grading instructions
4. Add example submission(s) and assessment(s) to exercise
5. Navigate to "Tutor Course Dashboard" via Course Administration
6. Click on "Exercise Dashboard" for the previously created exercise
7. Confirm reading the grading instructions
8. Review Example Submissions not "used for tutorial" 
9. Assess Example Submissions "used for tutorial"
10. See Students' Submissions appear
11. Assess Student Submission
